delfin/common/constants.py

```diff
@@ -169,6 +169,7 @@ class SecurityLevel(object):
 
 PROMETHEUS_EXPORTER_FILE = "/var/lib/delfin/delfin_exporter.txt"
 BLOCK_SIZE = 4096
+METRICS_SERVER_HOST = '0.0.0.0'
```

Comment:
We can move both of these in to delfin.conf . please check how 'kafka_port' anf 'kafka_ip'  is used

Reply:
@anvithks are these changes required? Please fix as per review comments

Reply:
@kumarashit Yes these changes are required. 
I do not understand the code enough to make the changes required to the conf file.
Request @NajmudheenCT or @PravinRanjan10 to look into this.

Reply:
I am looking in to this.. look like a restructure of conf parsing is rquired. we may have  to close this PR. I will update once I have some info.

Reply:
@NajmudheenCT sure. let me know how it goes.